src/procedures/procedures.11tydata.yaml

```
@@ -286,6 +286,186 @@ procedureMigrateZigbeeZHAOnly:
       image: z2m-backup-restore-05.png
       content: |
         <ul>
-        <li class="info">All migrated devices have a new entity ID. This means you will need to manually reconfigure automations and dashboard cards.</li>
+        <li class="info">All migrated devices have new entity IDs. This means you will need to manually reconfigure automations and dashboard cards.</li>
+          <li class="info">The names that you manually assigned to your devices won't have been migrated. You will have to rename the devices again.</li>
+        </ul>
+procedureMigrateConBee:
+  title: Migrate a deCONZ network to SkyConnect using ZHA
+  steps:
+    - title: View the firmware version of the stick
+      image: conbee-update-05.png
+      content: |
+        <ol>
+          <li>In the deCONZ app, under <b>Settings</b>><b>Gateway</b>, check the firmware version. It should be version 26720700.
+            <ul>
+              <li>If the version is correct, skip the next two steps and continue with the step on stopping the deCONZ add-on.</li>              
```

```suggestion
              <li>If the version is recent enough, skip the next two steps and continue with Step 4 <a href="#step-stop-the-deconz-add-on">Stop the deCONZ add-on</a>.</li>              
```

while this adds "maintenance" (because step numbering is implicit) etc, I think it's worth it. Lots of people will skip and can just click with this addition :smile: